docs/getting-started/cool-server.md

```diff
@@ -68,11 +68,11 @@ In this server, we implement many APIs and list their corresponding URLs as foll
 
   ```bash
   curl --location --request POST 'http://127.0.0.1:9009/cohort/selection' \
```

Comment:
replace the path on your machine with some pseudo paths. would looks nicer. Then we can merge it.

Reply:
ok